django_membersuite_auth/models.py

```diff
@@ -8,7 +8,8 @@
 @python_2_unicode_compatible
 class MemberSuitePortalUser(models.Model):
 
-    user = models.OneToOneField(settings.AUTH_USER_MODEL)
+    user = models.OneToOneField(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
Is this cool?  Do we really want, say, resources added to the hub when a MemberSuitePortalUser is deleted?